hadoop-tools/hadoop-azure-datalake/pom.xml

```diff
@@ -166,5 +166,12 @@
       <scope>test</scope>
       <type>test-jar</type>
     </dependency>
+
+    <dependency>
+      <groupId>org.wildfly.openssl</groupId>
+      <artifactId>wildfly-openssl</artifactId>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
why this one now given we have had this dependency forever but have not required this?

Reply:
2.3.9 version of SDK is expecting wildfly dependency.
without the same, nearly all the tests were failing with the following exception.
java.lang.NoClassDefFoundError: Could not initialize class com.microsoft.azure.datalake.store.SSLSocketFactoryEx

Reply:
is there some issue with your local build?

Because it should pull in hadoop-project's pom dependencies (https://github.com/apache/hadoop/blob/df35c7f51996b63a14a3323b40fb75a6f4c29c7d/hadoop-project/pom.xml#L1450) since gen1 driver pom does depend on hadoop-project pom.

Reply:
just because hadoop-project declares an artifact doesn't mean it comes in on the classpath. hadoop-common compiles with it but doesn't force it downstream, so modules which needed it do have to declare it.